src/components/shared/VirtualList.js

```diff
@@ -2,12 +2,28 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import React, { PureComponent } from 'react';
+// @flow
+import * as React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import range from 'array-range';
 
-class VirtualListRow extends PureComponent {
+import type { CssPixels } from '../../types/units';
+
+type RenderItem = (*, number, number) => React.Node;
+
+type VirtualListRowProps = {|
+  +renderItem: RenderItem,
+  +item: *,
+  +index: number,
+  +columnIndex: number,
+  +isSpecial: boolean,
+  // Items are not used directly, but are needed for strict equality checks so that
+  // the components update correctly.
+  +items: *,
```

Comment:
juse use `any` for this property

Reply:
I think you can still use `any` for this property `items` though: we don't care about its type.